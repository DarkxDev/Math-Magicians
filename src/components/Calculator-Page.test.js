import React from "react";
import renderer from "react-test-renderer";
import CalculatorPage from "./Calculator-Page";

test('Renders correctly', () => {
  const tree = renderer.create(<CalculatorPage />).toJSON();
  expect(tree).toMatchSnapshot();
})