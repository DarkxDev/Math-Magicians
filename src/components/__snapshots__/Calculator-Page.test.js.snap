// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div
  className="calculator-page"
>
  <h2>
    Let
    '
    s do some math!
  </h2>
  <div
    className="calculator"
  >
    <div
      className="display-row"
    >
      <input
        className="display"
        readOnly={true}
        type="text"
        value="0"
      />
    </div>
    <div
      className="button-grid"
    >
      <div
        className="upper-button-grid"
      >
        <button
          className="btn-grey button"
          onClick={[Function]}
          type="button"
        >
          AC
        </button>
        <button
          className="btn-grey button"
          onClick={[Function]}
          type="button"
        >
          +/-
        </button>
        <button
          className="btn-grey button"
          onClick={[Function]}
          type="button"
        >
          %
        </button>
        <button
          className="btn-orange button"
          onClick={[Function]}
          type="button"
        >
          รท
        </button>
        <button
          className="btn-grey button"
          onClick={[Function]}
          type="button"
        >
          7
        </button>
        <button
          className="btn-grey button"
          onClick={[Function]}
          type="button"
        >
          8
        </button>
        <button
          className="btn-grey button"
          onClick={[Function]}
          type="button"
        >
          9
        </button>
        <button
          className="btn-orange button"
          onClick={[Function]}
          type="button"
        >
          x
        </button>
        <button
          className="btn-grey button"
          onClick={[Function]}
          type="button"
        >
          4
        </button>
        <button
          className="btn-grey button"
          onClick={[Function]}
          type="button"
        >
          5
        </button>
        <button
          className="btn-grey button"
          onClick={[Function]}
          type="button"
        >
          6
        </button>
        <button
          className="btn-orange button"
          onClick={[Function]}
          type="button"
        >
          -
        </button>
        <button
          className="btn-grey button"
          onClick={[Function]}
          type="button"
        >
          1
        </button>
        <button
          className="btn-grey button"
          onClick={[Function]}
          type="button"
        >
          2
        </button>
        <button
          className="btn-grey button"
          onClick={[Function]}
          type="button"
        >
          3
        </button>
        <button
          className="btn-orange button"
          onClick={[Function]}
          type="button"
        >
          +
        </button>
      </div>
      <div
        className="lower-button-grid"
      >
        <button
          className="btn-grey button"
          onClick={[Function]}
          type="button"
        >
          0
        </button>
        <button
          className="btn-grey button"
          onClick={[Function]}
          type="button"
        >
          .
        </button>
        <button
          className="btn-orange button"
          onClick={[Function]}
          type="button"
        >
          =
        </button>
      </div>
    </div>
  </div>
</div>
`;
